Bottom: c03d8092be70f0261d133366c2198b89e0103b29
Top:    627006e0cdff81f449eafbdad98a53c03f630dca
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-07-04 00:12:16 +0100

hid/gerber: Fix broken eagle-suffix filename assignemnt

This was broken carelessly in commit c812fbae2dfe74cd3d39dd2dcc5cdbb08e07555a

NB: Please pay attention to compiler warnings... they are there for a reason!


---

diff --git a/src/hid/gerber/gerber.c b/src/hid/gerber/gerber.c
index 9fae772..29691bd 100644
--- a/src/hid/gerber/gerber.c
+++ b/src/hid/gerber/gerber.c
@@ -567,7 +567,7 @@ assign_file_suffix (char *dest, int idx)
     case NAME_STYLE_FIRST:  fns_style = FNS_first;  break;
     case NAME_STYLE_EAGLE:
       assign_eagle_file_suffix (dest, idx);
-      break;
+      return;
     case NAME_STYLE_HACKVANA:
       assign_hackvana_file_suffix (dest, idx);
       return;
